notebooks/tutorials/visualization/blender/landlab_to_blender.ipynb

```diff
@@ -0,0 +1,362 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid = HexModelGrid((nrows, ncols), spacing=node_spacing, node_layout=\"rect\")\n",
+    "grid.dx = (\n",
+    "    grid.spacing\n",
+    ")  # temporary hack to enable TaylorNonLinearDiffuser to work with hex grid\n",
+    "elev = grid.add_zeros(\"topographic__elevation\", at=\"node\")\n",
+    "elev[grid.core_nodes] = np.random.rand(grid.number_of_core_nodes)\n",
+    "K_sp = K_sp_mean + K_sp_std * np.random.randn(grid.number_of_nodes)\n",
+    "u = uplift_fn(grid.x_of_node, grid.y_of_node)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate three Landlab components: a `FlowAccumulator` to route flow across the gridded terrain, a `StreamPowerEroder` to simulate erosion by running water, and a `TaylorNonLinearDiffuser` to represent mass transport by gravitational processes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fa = FlowAccumulator(grid)\n",
+    "sp = StreamPowerEroder(grid, K_sp=K_sp)\n",
+    "td = TaylorNonLinearDiffuser(grid, linear_diffusivity=0.005)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Loop to simulate the evolution of the landscape."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in range(num_steps):\n",
+    "    elev[grid.core_nodes] += u[grid.core_nodes] * uprate * dt\n",
+    "    fa.run_one_step()\n",
+    "    sp.run_one_step(dt)\n",
+    "    td.run_one_step(dt)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Inspect the resulting terrain with an image color-coded to represent cell elevation values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imshow_grid(grid, elev)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Write the grid nodes and associated elevations to an `.obj` format file (note: the `topographic__elevation` field is the default choice for the $z$ coordinate of nodes)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "write_obj(\"test_mts.obj\", grid, clobber=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installing Blender\n",
+    "\n",
+    "The first step is to download and install the Blender application from [https://www.blender.org/download/](https://www.blender.org/download/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reading the output file into Blender\n",
```

Comment:
At the top of this section, it would be useful to have a screenshot of the Blender application with each section labeled with how you refer to them throughout the notebook. I.e., square highlights with labels around the "view pane," "view port," etc. (and maybe there's already something like this in the screenshots below, which I'm not seeing!)

Reply:
I think the screenshots should cover this